uring/src/main/scala/fs2/io/uring/Uring.scala

```diff
@@ -32,13 +32,13 @@ private[uring] final class Uring[F[_]](ring: UringExecutorScheduler)(implicit F:
 
   private[this] val noopRelease: Int => F[Unit] = _ => F.unit
 
-  def call(prep: Ptr[io_uring_sqe] => Unit): F[Int] =
-    exec(prep)(noopRelease)
+  def call(prep: Ptr[io_uring_sqe] => Unit, mask: Int => Boolean = _ => false): F[Int] =
+    exec(prep, mask)(noopRelease)
```

Comment:
We can cache the `_ => false` in a `private[this] val` like we do for `noopRelease`. Then we don't need to allocate it each time.